content/_data/changelogs/weekly.yml

```
@@ -17914,7 +17914,96 @@
   # pull: 7227 (PR title: Update dependency css-minimizer-webpack-plugin to v4.2.1)
   # pull: 7228 (PR title: Update dependency less-loader to v11.1.0)
   # pull: 7234 (PR title: Update dependency eslint to v8.25.0)
-  
+
+  - version: '2.374'
+    date: 2022-10-16
+    changes:
+
+      - type: rfe
+        category: rfe
+        pull: 7091
+        authors:
+          - omehegan
+        pr_title: Clarify safe restart won't wait for Pipeline jobs
+        message: |-
+          Clarify safe restart won't wait for Pipeline jobs.
+      - type: rfe
+        category: rfe
+        pull: 6951
+        authors:
+          - mawinter69
+          - basil
+        pr_title: Allow Formchecker to check more than 1 thing at a time
+        message: |-
+          Allow Formchecker to check more than one thing at a time.
```

"Formchecker" is not an English word.

```suggestion
          Allow form checker to check more than one thing at a time.
```

---

Ah, thanks for the heads up on this one @basil!